pycaw/tools/app_mixer/README.md

```diff
@@ -0,0 +1,14 @@
+# :sparkles: Python Live WASAPI Mixer
+
+## made with Python3, pycaw, and the use of Windows audio callbacks:
+
+_Demo:_
+https://user-images.githubusercontent.com/63397666/116756672-c5cfee00-aa0c-11eb-9cb5-c1f04e60acfa.mp4
+
+## controll and view live audio sessions:
```

Comment:
Typo s/controll/control/g Also in the commit body

Reply:
thanks for pointing that out ;)